modules/meeting/spec/models/project/activity_spec.rb

```diff
@@ -28,12 +28,12 @@
 
 require 'spec_helper'
 
-describe Projects::Activity, type: :model do
-  let(:project) do
-    create(:project)
+describe Projects::Activity, 'budget' do
```

Comment:
Probably a c&p oversight but should read 'meeting'

Reply:
Indeed. Thanks